client/rest-high-level/src/main/java/org/elasticsearch/client/ilm/RolloverAction.java

```diff
@@ -26,11 +26,12 @@
     private static final ParseField MAX_PRIMARY_SHARD_SIZE_FIELD = new ParseField("max_primary_shard_size");
     private static final ParseField MAX_AGE_FIELD = new ParseField("max_age");
     private static final ParseField MAX_DOCS_FIELD = new ParseField("max_docs");
+    private static final ParseField MAX_SHARD_DOCS_FIELD = new ParseField("max_shard_docs");
```

Comment:
I don't think it is needed to add this new limit to the HLRC code base. 
This HLRC is no longer developed in favour for the new java client.
This just exists in master/8.0 for tests that still rely on hlrc, 
but there should be no need to add new things to the HLRC.
The HLRC is no longer released / published and will be removed in the near future.
So I think we can undo all changes in the client directory.

Reply:
got it, I will revert them.